s3/pkg/utils/utils.go

```diff
@@ -151,8 +158,3 @@ func SetRepresentTenant(ctx context.Context, requestTenant, sourceTenant string)
 
 	return ctx
 }
-
```

Comment:
THanks, I fixed it by adding those lines back. It is not used anywhere, may be it will be used in near future, I added it back. Same constants are defined in common.go also.